cmd/go-filecoin/payment_channel_daemon_test.go

```diff
@@ -269,49 +265,84 @@ func TestPaymentChannelReclaimSuccess(t *testing.T) {
 	channelExpiry := types.NewBlockHeight(3).Add(bh)
 	channelAmount := types.NewAttoFILFromFIL(1000)
 
-	balanceBefore, err := rsrc.payer.WalletBalance(ctx, rsrc.payerAddr)
+	balanceBefore, err := rsrc.payer.PorcelainAPI.WalletBalance(ctx, rsrc.payerAddr)
 	require.NoError(t, err)
 
 	chanid, gasReceipt := rsrc.requirePaymentChannel(ctx, t, channelAmount, channelExpiry)
 
 	voucherAmount := types.NewAttoFILFromFIL(10)
 	voucherValidAt := types.NewBlockHeight(0)
-	voucherStr, err := rsrc.payer.PaychVoucher(ctx, chanid, voucherAmount, fast.AOFromAddr(rsrc.payerAddr), fast.AOValidAt(voucherValidAt))
+
+	voucher, err := rsrc.payer.PorcelainAPI.PaymentChannelVoucher(ctx, rsrc.payerAddr, chanid, voucherAmount, voucherValidAt, nil)
 	require.NoError(t, err)
 
-	mcid, err := rsrc.target.PaychRedeem(ctx, voucherStr, fast.AOFromAddr(rsrc.targetAddr), fast.AOPrice(big.NewFloat(1)), fast.AOLimit(300))
+	params := []interface{}{
+		voucher.Payer,
+		&voucher.Channel,
+		voucher.Amount,
+		&voucher.ValidAt,
+		voucher.Condition,
+		[]byte(voucher.Signature),
+		[]interface{}{},
+	}
+	mcid, _, err := rsrc.target.PorcelainAPI.MessageSend(
+		ctx,
+		rsrc.targetAddr,
+		address.PaymentBrokerAddress,
+		types.NewAttoFILFromFIL(0),
+		types.NewAttoFIL(big.NewInt(1)),
+		300,
+		paymentbroker.Redeem,
+		params...,
+	)
```

Comment:
I think it's probably worth factoring out a wrapper or sender object to reduce the verbosity of message sends. Either a free function with parameters similar to the FAST method this used to have, or something that curries in parameters like the sender and gas price/limit.